helper/src/main/java/io/github/libxposed/helpers/ModifierHelper.java

```diff
@@ -0,0 +1,154 @@
+package io.github.libxposed.helpers;
+
+import java.lang.reflect.Member;
+import java.lang.reflect.Modifier;
+
+/**
+ * Helper for quick check the modifier of a member/class.
+ */
+@SuppressWarnings("unused")
+public class ModifierHelper {
```

Comment:
create a private conatructor for it and throw expection in it to prevent instanization